src/app/core/learning-object-module/learning-object/learning-object.routes.ts

```diff
@@ -122,6 +137,28 @@ export const LEARNING_OBJECT_ROUTES = {
     DELETE_LEARNING_OBJECT(learningObjectId: string) {
         return `${environment.apiURL}/learning-objects/${encodeURIComponent(learningObjectId)}`;
     },
+
+    POST_MAPPING(username: string, learningObjectId: string, outcomeId: string) {
+        return `${environment.apiURL}/users/${encodeURIComponent(
+            username,
+        )}/learning-objects/${encodeURIComponent(
+            learningObjectId,
+        )}/outcomes/${encodeURIComponent(outcomeId)}/mappings`;
+    },
+    DELETE_MAPPING(
+          username: string,
+          learningObjectId: string,
+          outcomeId: string,
+          mappingsId: string,
+      ) {
+          return `${environment.apiURL}/users/${encodeURIComponent(
```

Comment:
I'm pretty sure this all got updated and should be in the learning-object-module/outcomes module. There aren't any routes called /mappings in the gateway or backend

![Screenshot 2024-09-09 at 9 37 20 AM](https://github.com/user-attachments/assets/3aefa183-fb2d-4e0e-b519-74caa260082e)